hw1/02-palindrome.js

```diff
@@ -1,3 +1,45 @@
 const elem = document.querySelector('input');
+const errorElem = document.querySelector('.inputError');
+const resultElem = document.querySelector('.displayResult');
 
 elem.addEventListener('input', handleInput);
```

Comment:
Airbnb Linter gave this message:
   5:32  error  'handleInput' was used before it was defined   no-use-before-define